test/Interop/SwiftToCxx/class/swift-actor-in-cxx.swift

```diff
@@ -0,0 +1,44 @@
+// RUN: %empty-directory(%t)
+// RUN: %target-swift-frontend %s -typecheck -module-name Actor -clang-header-expose-decls=has-expose-attr -emit-clang-header-path %t/actor.h -disable-availability-checking
+// RUN: %FileCheck %s < %t/actor.h
+
+// RUN: %check-interop-cxx-header-in-clang(%t/actor.h)
+
+// RUN: %target-swift-frontend %s -typecheck -module-name Actor -enable-library-evolution -clang-header-expose-decls=has-expose-attr -emit-clang-header-path %t/actor-evo.h -disable-availability-checking
+// RUN: %FileCheck %s < %t/actor-evo.h
+
+// RUN: %check-interop-cxx-header-in-clang(%t/actor-evo.h)
+
+// REQUIRES: concurrency
+
+@_expose(Cxx)
+public final actor ActorWithField {
+  var field: Int64
+
+  public init() {
+    field = 0
+    print("init ActorWithField")
+  }
+  deinit {
+    print("destroy ActorWithField")
+  }
+
+  public func isolatedMethod() {}
+
+  public nonisolated func method() {
+      print("nonisolated method")
+  }
```

Comment:
I wonder if you'd also want to add a 

```
public nonisolated func methodAsync() async 
```

and confirm it does not import; though I guess this perhaps is already handled -- it's the same as any normal async function after all.

Reply:
async functions are not bridged already, we cover that. But I can add as a follow-up!